index.html

```diff
@@ -1,20 +1,320 @@
+            </div>
+          </div>
+          <div
+            class="teams-card-col d-flex gap-4 col-sm-12 col-md-12 col-lg-6 pb-1"
+          >
+            <img
+              src="https://source.unsplash.com/random"
+              class="col-sm-4 col-md-4 col-lg-5"
+              alt="card-img"
+            />
+            <div
+              class="teams-card-txt d-flex flex-column gap-3 col-sm-6 col-md-6 col-lg-5"
+            >
+              <p class="fw-bold fs-4 text-dark">John Doe</p>
+              <p class="fst-italic">Director, Chairman hevons Farms</p>
+              <p class="fs-small mt-1 text-dark">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit
+                non sapiente in.
+              </p>
+            </div>
+          </div>
+          <div
+            class="teams-card-col d-flex gap-4 col-sm-12 col-md-12 col-lg-6 pb-1"
+          >
+            <img
+              src="https://source.unsplash.com/random"
+              class="col-sm-4 col-md-4 col-lg-5"
+              alt="card-img"
+            />
+            <div
+              class="teams-card-txt d-flex flex-column gap-3 col-sm-6 col-md-6 col-lg-5"
+            >
+              <p class="fw-bold fs-4 text-dark">John Doe</p>
+              <p class="fst-italic">Director, Chairman hevons Farms</p>
+              <p class="fs-small mt-1 text-dark">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit
+                non sapiente in.
+              </p>
+            </div>
+          </div>
+        </div>
+        <button
+          type="button"
+          class="btn btn-outline-warning w-75 mx-auto text-warning p-4 d-lg-none d-flex align-items-center justify-content-center gap-4"
+        >
+          More
+          <span class="material-icons icons"> keyboard_arrow_down </span>
+        </button>
+      </section>
+      <!-- Partners Section -->
+      <section
+        class="bg-dark d-lg-flex flex-column gap-4 col-12 py-3 d-none partners"
+      >
+        <h3 class="h3 text-center">Our Partners</h3>
+        <div class="line"></div>
+        <div class="d-flex justify-content-evenly col-12">
+          <img src="./img/bank.png" alt="bankIcon" />
+          <img src="./img/insurance.png" alt="insuranceIcon" />
+          <img src="./img/google.webp" alt="googleIcon" />
+        </div>
+      </section>
+    </main>
+    <footer class="bg-light d-none d-lg-flex gap-4 col-12 py-3 px-5 d-none">
+      <div class="col-5 py-3">
+        <img src="./img/logo.png" class="w-25" alt="logo" />
+        Hevons Synergy
+      </div>
+      <div class="col-6 px-2 py-3">
+        &copy; 2022, Some rights reserved
+        <p class="fst-itealic fs-small fw-light">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi
+          voluptatem sapiente, dolor quia error numquam blanditiis voluptatibus
+          quam? Qui optio vitae ad molestias expedita consectetur!
+        </p>
```

Comment:
- [ ] For the footer, kindly align your items to the center. Right now it looks as if the text is above the logo. It would be great if you could address this issue.
- Your design:
![Capture6](https://user-images.githubusercontent.com/85113567/181270243-9163575e-751d-43d9-a8e3-ac8b4f47a263.PNG)